explainability-service/src/main/java/org/kie/trustyai/service/prometheus/PrometheusScheduler.java

```diff
@@ -43,6 +43,13 @@ public Map<UUID, BaseMetricRequest> getSpdRequests() {
         return spdRequests;
     }
 
+    public Map<UUID, BaseMetricRequest> getAllRequests() {
+        // extend this with other metrics when more are added
+        Map<UUID, BaseMetricRequest> result = getDirRequests();
+        result.putAll(getSpdRequests());
```

Comment:
I don't think you want to reuse/modify the `dirRequests` map
```suggestion
        Map<UUID, BaseMetricRequest> result = new HashMap<>();
        result.putAll(getDirRequests());
        result.putAll(getSpdRequests());
```